geomesa-tools/src/main/scala/org/locationtech/geomesa/tools/FeatureExporter.scala

```diff
@@ -93,6 +93,46 @@ object ShapefileExport {
 
     modifiedAttrs.mkString(",")
   }
+
+  def replaceGeomInAttributesString(attributes: String, sft: SimpleFeatureType): String = {
+     /**
+      * Check that:
+      * 1. The geometry descriptor is not called "the_geom" so we don't replace that.
+      * 2. The geometry descriptor is contained in the attributes
+      * 3. Immediately before AND after the geometry descriptor is undefined (geom query only)
+      *    OR immediately before is undefined AND after is comma (geom first in list)
+      *    OR immediately before is a comma AND after is undefined (geom last in list)
+      *    OR comma before AND after geom (geom in middle of list)
+      * 4. The geometry descriptor isn't being transformed already (even if the transformation is
+      *    incorrect, as we'll take care of that at a later stage in the QueryPlanner)
+      */
+    val trimmedAttributes = attributes.split(",").map(_.trim).mkString(",")
```

Comment:
I think there is some logic somewhere for parsing out this attribute string that will allow for escaping commas - we should use that here.